implement the passthrough importer for all resources

displague opened the pull request:
Fixes #49 

https://www.terraform.io/docs/extend/resources/import.html

sarahhodne commented:
I'm curious, is there a reason why the importer wasn't also implemented for `equinix_network_device`?

displague commented:
Thanks, @sarahhodne. I must have overlooked that file. The ID appears to be importable in that resource too.  Added.

I've also added the import documentation for these resources.
